.github/scripts/generate_binary_build_matrix.py

```diff
@@ -96,6 +95,10 @@ def arch_type(arch_version: str) -> str:
 
 # This can be updated to the release version when cutting release branch, i.e. 2.1
 DEFAULT_TAG = "main"
+release_version = os.getenv("RELEASE_VERSION_TAG")
+# if release_version is specified, use it to validate the packages
+if(release_version):
```

Comment:
(This is python, not a C++)
```suggestion
if release_version :
```